docs/simulating_with_the_mvs.html

```diff
@@ -0,0 +1,679 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
why did you add this file here @mahendrark ?

Reply:
Woah, I do not remember adding this file. Not sure how it crept in! gitignore issue?

Reply:
Use a visual git interface (like git cola) to avoid these inserts ;)
Remove the file and see if the docs still works without it (I guess it should)

Reply:
got it
removed the line and pushed the change now